components/config/prefs.rs

```diff
@@ -216,7 +216,6 @@ pub struct Preferences {
     pub js_mem_max: i64,
     pub js_native_regex_enabled: bool,
     pub js_offthread_compilation_enabled: bool,
-    pub js_parallel_parsing_enabled: bool,
```

Comment:
Firefox still has this pref for enabling offthread : https://searchfox.org/firefox-main/search?q=parallel_parsing&path=&case=false&regexp=false, not saying that we should keep it tho.

Reply:
No need for us to keep preferences that aren't actually hooked up to anything.